.ci/pitest.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on CircleCI
+set -e
+
+case $1 in
+
+pitest-annotation \
+|pitest-ant \
+|pitest-api \
+|pitest-blocks \
```

Comment:
Can these not be hard coded and derived from `pom.xml`

Reply:
I made bash more smart.

Reply:
Resolved